src/main/java/dev/thedocruby/resounding/effects/math/Air.java

```diff
@@ -1,15 +1,15 @@
-package dev.thedocruby.resounding.effects;
+package dev.thedocruby.resounding.effects.math;
```

Comment:
Since you are doing modular, the air class should be separated into Biome effects (temp/humididy) and rain effects. Dont worry about the actuall effect code, just make blank ones like the other effects. I never properly rewrote this, it shouldn't be in the mod this early anyway. Just comment out the class body so we still have the code just in case. Also, be sure to `exclude` the related settings in the config.

Reply:
Done.